src/App/App.test.js

```diff
@@ -1,24 +1,175 @@
-import { render, screen, fireEvent } from '@testing-library/react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event'
 import '@testing-library/jest-dom';
 import App from './App';
+import { getAllMoviesData, getSingleMovieData } from '../apiCalls';
+jest.mock('../apiCalls.js');
+
 // no need to import other components for integration testing as app already has those components imported and the child components that were imported into its children
 
 describe('App', () => {
-  it('should take user to movie cards view when return to home button is clicked', () => {
-    // setup
-    // render app with all components, with singleMovie displayed and mocked returntohome
-    // execution
-    // fireEvent or userEvent for onclick of Return to home button
-    // assertion
-    // expect movieContainer to show cards
+  it('should dispay movie cards on page load', async () => {
+    getAllMoviesData.mockResolvedValueOnce(
+      {'movies': [
```

Comment:
So our typo was in our mocked movies?

Reply:
It was in mocked movies data for a single movie.